modules/social_features/social_search/src/EventSubscriber/RedirectSubscriber.php

```diff
@@ -103,7 +103,7 @@ public function redirectSearchWithPrefilledExposedFilters(RequestEvent $event) {
 
     // Workaround for drupal.org issue #3085806.
     if (empty($query) || empty($query['created_op'])) {
-      $query = ['created_op' => '<'];
+      $query = array_merge($query, ['created_op' => '<']);
```

Comment:
$query is null and has no value here. Array merge will raised notice for providing null instead of array.

Reply:
Updated.